### PR TITLE
Remove auth plugin and all references

### PR DESCRIPTION
**Pull request by darrellwarde**
# Description

Remove auth plugin and all references.

The only sticking point was `neo-place` - I've honestly lost track of the status here, but I've blindly updated it for now.

## Complexity

Complexity: Low

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-3662.surge.sh

**Comment by neo4j-team-graphql** [comment]
# Performance Report

No Performance Changes


<details>
    <summary>Show Full Table</summary>
    
| name | dbHits | old dbHits | time (ms) | old time (ms) | maxRows |
| ---- | ------ | ---------- | --------- | ------------- | ------- |
|  aggregations.TopLevelAggregate | 3403 | 3403 | 52 | 59 | 1134 |
|  aggregations.NestedAggregation | 16553 | 16553 | 93 | 129 | 2174 |
|  aggregations.AggregationWithWhere | 11979 | 11979 | 59 | 66 | 2174 |
|  aggregations.AggregationWhereWithinNestedRelationships | 22116987 | 22116987 | 4452 | 4033 | 2008534 |
|  aggregations.AggregationWhereWithinNestedConnections | 22116987 | 22116987 | 3911 | 4103 | 2008534 |
|  aggregations.NestedCountFromMovieToActors | 9734 | 9734 | 116 | 75 | 2174 |
|  aggregations.NestedCountFromActorsToMovie | 9937 | 9937 | 97 | 81 | 2174 |
|  aggregations.DeeplyNestedCount | 13070353 | 13070329 | 5862 | 6386 | 2008534 |
|  batch-create.BatchCreate | 4200 | 4200 | 205 | 219 | 600 |
|  batch-create.BatchCreateSmall | 77 | 77 | 99 | 135 | 11 |
|  connect.createAndConnect | 14424 | 14424 | 294 | 229 | 3003 |
|  connections.Connection | 14082 | 14082 | 89 | 99 | 2174 |
|  connections.NestedConnection | 45472 | 45472 | 134 | 153 | 4516 |
|  connections.ConnectionWithSort | 3284 | 3284 | 85 | 100 | 1040 |
|  connections.ConnectionWithSortAndCypher | 16325 | 16325 | 115 | 142 | 2174 |
|  create.SimpleMutation | 7 | 7 | 73 | 89 | 1 |
|  cypher-directive.TopLevelMutationDirective | 1135 | 1135 | 40 | 40 | 1134 |
|  delete.SimpleDelete | 19401 | 19401 | 343 | 352 | 1040 |
|  delete.NestedDeleteInUpdate | 19542 | 19542 | 254 | 293 | 1179 |
|  2925.SingleRelationshipFilter | 6468 | 6468 | 60 | 62 | 1040 |
|  2925.NestedSingleRelationshipFilter | 22900 | 22900 | 111 | 164 | 2174 |
|  2925.SingleRelationshipRequiredFilter | 5201 | 5201 | 56 | 56 | 1040 |
|  2925.NestedSingleRelationshipRequiredFilter | 9361 | 9361 | 75 | 102 | 1040 |
|  query.SimpleQuery | 3121 | 3121 | 31 | 30 | 1040 |
|  query.SimpleQueryWithRelationship | 16162 | 16162 | 59 | 69 | 2174 |
|  query.QueryWhere | 9695 | 9715 | 51 | 53 | 2161 |
|  query.SimpleQueryWithNestedWhere | 9873 | 9893 | 71 | 108 | 2161 |
|  query.Nested | 10096041 | 10096041 | 12578 | 11645 | 2008534 |
|  query.NestedWithFilter | 10074401 | 10074401 | 12505 | 11851 | 2004000 |
|  query.OrFilterOnRelationships | 43741 | 44122 | 264 | 278 | 2081 |
|  query.OrFilterOnRelationshipsAndNested | 37679 | 37822 | 321 | 329 | 2081 |
|  query.QueryWithNestedIn | 13707 | 14969 | 75 | 84 | 1320 |
|  query.NestedConnectionWhere | 9834 | 9834 | 76 | 96 | 2174 |
|  query.DeeplyNestedConnectionWhere | 9964 | 10054 | 109 | 129 | 2174 |
|  query.DeeplyNestedWithRelationshipFilters | 19158 | 19344 | 177 | 297 | 1602 |
|  query.NestedWithRelationshipSingleFilters | 3881 | 3881 | 219 | 474 | 1134 |
|  query.Fulltext | 80 | 80 | 56 | 45 | 16 |
|  query.FulltextWithNestedQuery | 587 | 587 | 67 | 60 | 84 |
|  sorting.SortMultipleTypes | 3515 | 3515 | 123 | 88 | 1040 |
|  sorting.SortMultipleTypesWithCypherWithCypher | 14532 | 14467 | 204 | 125 | 2174 |
|  sorting.SortOnNestedFields | 14082 | 14082 | 85 | 73 | 2174 |
|  sorting.SortDeeplyNestedFields | 43198 | 43198 | 158 | 144 | 4516 |
|  sorting.SortWithTopLevelCypher | 16162 | 16162 | 69 | 91 | 2174 |
|  unions.SimpleUnionQuery | 321 | 321 | 78 | 94 | 35 |
|  unions.SimpleUnionQueryWithMissingFields | 293 | 293 | 87 | 88 | 35 |
|  unions.NestedUnion | 410637 | 410637 | 373 | 339 | 33033 |
|  unions.NestedUnionWithMissingFields | 384611 | 384611 | 351 | 345 | 33033 |
|  update.NestedUpdate | 16143 | 16143 | 164 | 282 | 2002 |

</details>
        
Old Schema Generation: 59.873s
Schema Generation: 57.878s
Old Subgraph Schema Generation: 1:13.515 (m:ss.mmm)
Subgraph Schema Generation: 1:12.315 (m:ss.mmm)

File: docs/modules/ROOT/pages/auth/index.adoc [hunk 1]
```diff
@@ -1,51 +0,0 @@
-[[auth]]
```
Comment (tbwiss) [inline docs/modules/ROOT/pages/auth/index.adoc:1]: (It's not here)
We probably also need to clean up the `content-nav.adoc` file

**Review by tbwiss — APPROVED** [review]
LGTM!

File: docs/modules/ROOT/pages/auth/index.adoc (hunk 1 above)
Comment (darrellwarde) [reply docs/modules/ROOT/pages/auth/index.adoc:1]: Great spot, thank you!

**Pull request merged**